src/core/MIGRATION.md

```diff
@@ -132,17 +140,17 @@ The platform does not impose any technical restrictions on how the internals of
 
 The various independent domains that make up `core` are represented by a series of services, and many of those services expose public interfaces that are provided to _all_ plugins. Services expose different features at different parts of their _lifecycle_. We describe the lifecycle of core services and plugins with specifically-named functions on the service definition.
 
-In the new platform, there are two lifecycle functions today: `setup` and `stop`. The `setup` functions are invoked sequentially while Kibana is starting up on the server or when it is being loaded in the browser. The `stop` functions are invoked sequentially while Kibana is gracefully shutting down on the server or when the browser tab or window is being closed.
+In the new platform, there are three lifecycle functions today: `setup`, `start`, and `stop`. The `setup` functions are invoked sequentially while Kibana is starting up on the server or when it is being loaded in the browser. The `start` functions are invoked sequentially after setup has completed for all plugins. The `stop` functions are invoked sequentially while Kibana is gracefully shutting down on the server or when the browser tab or window is being closed.
```

Comment:
probably
```
The `setup` functions are invoked sequentially while Kibana is bootstrapping the server ...
```
also how about to make a small table

lifecycle event | server | client
--- | --- | ---
setup | bootstrapping the server| app is being loaded in the browser
start | ...| ...
stop | ...| ...

Reply:
Agreed, this is confusing. I like the table idea, I'll add that!